test/controllers/likes_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```

Comment:
If you don't have to need this test controller so why this is here ?

Reply:
Okay next time i will try not to add these files to commit